src/code/views/node-view.coffee

```diff
@@ -12,6 +12,9 @@ NodeTitle = React.createFactory React.createClass
   displayName: "NodeTitle"
   mixins: [require '../mixins/node-title']
 
+  componentWillMount: ->
+    @canDeleteWhenEmpty = @props.node.addedThisSession and not @props.node.titleUpdated
```

Comment:
> on subsequent edits of the title (which mounts this component)

node-view gets re-mounted when you edit the title? When testing, it doesn't seem to, so `@props.node.titleUpdated` looks like it can only ever be `undefined` at this point. But maybe I'm missing something.